jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/TraceEnrichingFilter.java

```diff
@@ -91,4 +94,23 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
             return Optional.of(header.equals("1"));
         }
     }
+
+    private static void addUserIdMetadataToSpan(ContainerRequestContext requestContext) {
+        Optional<String> userId = tryGetUnverifiedUserId(requestContext);
+        if (userId.isPresent()) {
+            Tracer.addSpanMetadata(ImmutableMap.of(BearerTokenLoggingFilter.USER_ID_KEY, userId.get()));
+        }
+    }
+
+    private static Optional<String> tryGetUnverifiedUserId(ContainerRequestContext requestContext) {
+        String userIdKey = BearerTokenLoggingFilter.getRequestPropertyKey(BearerTokenLoggingFilter.USER_ID_KEY);
+        Object maybeUserId = requestContext.getProperty(userIdKey);
```

Comment:
BearerTokenLoggingFilter already has `@Priority(Priorities.AUTHORIZATION)`. this is lower (i.e. higher priority) than the default value of Priorities.USER, so I think we're ok here.